src/Microsoft.Tye.Core/KubernetesManifestGenerator.cs

```diff
@@ -374,8 +374,9 @@ internal static class KubernetesManifestGenerator
                         {
                             var containerPort = new YamlMappingNode();
                             ports.Add(containerPort);
-                            containerPort.Add("containerPort", binding.Port.Value.ToString());
+                            containerPort.Add("containerPort", (binding.ContainerPort ?? binding.Port.Value).ToString());
                         }
+
```

Comment:
nit: new line not needed.

Reply:
🤨

Reply:
thx, that will teach me to double check when editing directly from the portal :D
(Not on the head please)